modules/pxp-module-puppet.md

```diff
@@ -0,0 +1,212 @@
+# A PXP module for kicking puppet runs
+
+## Installation
+
+Place `modules/pxp-module-puppet` in your pxp-agent's `modules-dir`. This can either be
+configured in pxp-agent's configuration file, by default located in
+`/etc/puppetlabs/pxp-agent/pxp-agent.conf`, or by starting the agent from the
```

Comment:
Should we mention the windows default value (which is - or at least was - different) as well.